_viash.yaml

```
@@ -0,0 +1,19 @@
+viash_version: 0.9.0-RC4
+
+name: <task_template>
+description: |
+  An OpenProblems benchmark on single-cell cell-cell communication.
+license: MIT
+keywords: [single-cell, cell-cell communication, openproblems, benchmark]
+links:
+  issue_tracker: https://github.com/openproblems-bio/<task_template>/issues
+  repository: https://github.com/openproblems-bio/<task_template>
+  docker_registry: ghcr.io/openproblems-bio
+
+config_mods: |
+  .version := 'dev'
+  .arguments[.multiple == true].multiple_sep := ';'
+  .runners[.type == "nextflow"].directives.tag := "$id"
+  .runners[.type == "nextflow"].auto.simplifyOutput := false
+  .runners[.type == "nextflow"].config.labels := { lowmem : "memory = 20.Gb", midmem : "memory = 50.Gb", highmem : "memory = 100.Gb", lowcpu : "cpus = 5", midcpu : "cpus = 15", highcpu : "cpus = 30", lowtime : "time = 1.h", midtime : "time = 4.h", hightime : "time = 8.h", veryhightime : "time = 24.h" }
+  .runners[.type == "nextflow"].config.script := "process.errorStrategy = 'ignore'"
```

```suggestion
version: dev

config_mods: |
  .runners[.type == "nextflow"].config.labels := { lowmem : "memory = 20.Gb", midmem : "memory = 50.Gb", highmem : "memory = 100.Gb", lowcpu : "cpus = 5", midcpu : "cpus = 15", highcpu : "cpus = 30", lowtime : "time = 1.h", midtime : "time = 4.h", hightime : "time = 8.h", veryhightime : "time = 24.h" }
```

Most of these can be omitted.